pkg/tenant/db/users.go

```diff
@@ -21,9 +21,15 @@ func CreateUserResources(ctx context.Context, orgName string, member *Member) (e
 	// New user should have a tenant
 	tenant := &Tenant{
 		OrgID:           member.OrgID,
-		Name:            slug.Make(orgName),
 		EnvironmentType: "development",
 	}
+
+	if member.Workspace == "" {
+		tenant.Name = slug.Make("dev-" + member.Workspace)
+	} else {
+		tenant.Name = "dev-tenant"
+	}
```

Comment:
Since we no longer have the organization name on Register, I changed this to use the workspace/domain name.